src/lib/event.ts

```diff
@@ -193,18 +195,23 @@ const getAllEvents = async (): Promise<Event> => {
 const getEvent = async (pid: number): Promise<EventWithCid> => {
   const event = await getObject(`${listKey}:pid:${pid}`);
   const cid = await getCidByPid(event.pid);
+  const fixedEvent = fixEvent(event);
 
   return {
-    ...fixEvent(event),
+    ...fixedEvent,
+    responsesCount: await getResponsesCount({ pid: +event.pid }),
     cid,
   };
 };
 
 const getEventsEndingAfter = async (endDate: number): Promise<Event[]> => {
   const keys = await getSortedSetRangeByScore(listByEndKey, 0, -1, endDate, +Infinity);
-  const events = (await getObjects(keys)).filter(Boolean);
+  const events: JsonEvent[] = (await getObjects(keys)).filter(Boolean);
 
-  return events.map(fixEvent);
+  return Promise.all(events.map(fixEvent).map(async event => ({
```

Comment:
I don't think you need to modify this function, just `getEvent` and `getEventsByDate`. I may be wrong though

Reply:
Yeah go ahead and remove it for now. Maybe add a comment documenting that responseCounts are not returned with this method.

Reply:
`getAllEvents` is missing the `responseCount`, too. Should I add it to both or now?

Reply:
added a comment to `allEvents`, too.